rust_snuba/src/consumer.rs

```diff
@@ -215,6 +215,10 @@ pub fn consumer_impl(
         })
         .collect();
 
+    if let Some(group_instance_id) = consumer_config.group_instance_id {
+        broker_config.insert("group.instance.id".to_owned(), group_instance_id);
+    }
+
```

Comment:
Can you have the `resolve_consumer_config()` function handle this? It's supposed to be the singular place where consumer config is resolved before passing it to the Rust or Python consumer so we can avoid needing to do stuff like this twice in Rust and Python

Reply:
I can do it on the python side, but I still need to a way to get that into `KafkaConfig::new_consumer_config`? Should I make `group_instance_id` a parameter to that function? Here I resolved to just adding it into the overrides hashmap.